CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## Pelion Edge 2.2 - January 2021
+
+The primary features in this release:
+* [edge-core] Updated Edge Core to [0.15.0](https://github.com/PelionIoT/mbed-edge/blob/master/CHANGELOG.md#release-0150-2021-1-12)
+  * The new FOTA update framework library will be supported on platforms - `imx8mmevk` and `uz3eg-iocc`, (Feb 2021 release) not on `raspberrypi3`. To compile with this library, add following Bitbake parameters to local.conf - `MBED_EDGE_CORE_CONFIG_FIRMWARE_UPDATE="ON"`, `MBED_EDGE_CORE_CONFIG_FOTA_ENABLE="ON"` and `MBED_EDGE_CORE_CONFIG_CURL_DYNAMIC_LINK="ON"`.
```

Comment:
The new FOTA update framework is supported on platforms - imx8 and Xilinux board which are expected to release in Feb 2021. RPi do not support that in the coming release.

Reply:
```suggestion
  * The new FOTA update framework library will be supported on platforms - `imx8mmevk` and `uz3eg-iocc`, but not on `raspberrypi3`. To compile with this library, add following Bitbake parameters to local.conf - `MBED_EDGE_CORE_CONFIG_FIRMWARE_UPDATE="ON"`, `MBED_EDGE_CORE_CONFIG_FOTA_ENABLE="ON"` and `MBED_EDGE_CORE_CONFIG_CURL_DYNAMIC_LINK="ON"`.
```

Reply:
Gotcha. Adding a suggestion since it seems like information is duplicated?
(The header ahready has the release date, unless were going to do another release next month???)